packages/vc-api-verifier/README.md

```diff
@@ -29,6 +29,22 @@ yarn build
 ```
 
 ### Test
+To run the tests a `VC_HTTP_API_AUTH_TOKEN` environment variable needs to be present with an authorization token.
+
+#### Environment Variable:
```

Comment:
These are realy specific notes that will certainly not apply to all environment. I think it would even confuse people. We shouldn't be explaining how to set environment variables, just that they are needed